packages/amazon-ecs-cni-plugins/amazon-ecs-cni-plugins.spec

```
@@ -0,0 +1,103 @@
+%global ecscni_goproject github.com/aws
+%global ecscni_gorepo amazon-ecs-cni-plugins
+%global ecscni_goimport %{ecscni_goproject}/%{ecscni_gorepo}
+%global ecscni_gitrev 53a8481891251e66e35847554d52a13fc7c4fd03
+
+# Construct reproducible tar archives
+# See https://reproducible-builds.org/docs/archives/
+%global source_date_epoch 1234567890
+%global tar_cf tar --sort=name --mtime="@%{source_date_epoch}" --owner=0 --group=0 --numeric-owner -cf
+
+Name: %{_cross_os}amazon-ecs-cni-plugins
+Version: %{ecscni_gitrev}
+Release: 1%{?dist}
+Summary: Networking Plugins for ECS Task Networking
```

nit: no need to title case the summary
```suggestion
Summary: Networking plugins for ECS task networking
```